main.js

```diff
@@ -31,19 +31,38 @@ define(function (require, exports, module) {
     
     // --- Required modules ---
 
-    var CommandManager     = brackets.getModule("command/CommandManager");
-    var EditorManager      = brackets.getModule("editor/EditorManager");
-    var ExtensionUtils     = brackets.getModule("utils/ExtensionUtils");
-    var Menus              = brackets.getModule("command/Menus");
-    var PreferencesManager = brackets.getModule("preferences/PreferencesManager");
-    var CodeMirror = brackets.getModule("thirdparty/CodeMirror2/lib/codemirror");
+    var _                  = brackets.getModule("thirdparty/lodash"),
+        AppInit            = brackets.getModule("utils/AppInit"),
+        CodeMirror         = brackets.getModule("thirdparty/CodeMirror2/lib/codemirror"),
```

Comment:
CodeMirror is an unused variable.  I don't know how I missed this one earlier ;)